clientconn.go

```diff
@@ -1177,14 +1218,11 @@ func (ac *addrConn) createTransport(backoffNum int, addr resolver.Address, copts
 		// We don't want to reset during this close because we prefer to kick out of this function and let the loop
```

Comment:
This comment is for `close(skipReset)` not the `newTr.Close()`. But this comment is unnecessary anyway since ac is in SHUTDOWN state where no reconnecting will ever happen.